tests/rptest/services/admin.py

```diff
@@ -180,16 +170,3 @@ def _has_leader():
         leader = self.redpanda.get_node(details['leader_id'])
         ret = self._request('post', path=path, node=leader)
         return ret.status_code == 200
-
-    def _request_to_any(self, handler):
-        def try_send():
-            nodes = [n for n in self.redpanda.started_nodes()]
-            random.shuffle(nodes)
-            for node in nodes:
-                if handler(node):
-                    return True
-
-        wait_until(try_send,
-                   timeout_sec=30,
-                   backoff_sec=2,
-                   err_msg="Failed to complete request")
```

Comment:
aren't retries still needed at the client level for the cases when we hit the admin api at a moment when there is no known leader and get a error return rather than a redirect?

Reply:
Yep, retries are still needed, but only on 503 and connection errors -- that's handled in improvements to the Admin._request function in a later commit.